src/renderer/components/layout/BreadCrumb.tsx

```diff
@@ -45,22 +45,26 @@ class BreadCrumb extends React.Component<IProps> {
                         <SVG svg={ArrowIcon} />
                     </Link>
                 }
-                {breadcrumb && breadcrumb.map((item, index) =>
-                    item.path && index !== breadcrumb.length - 1 ?
+                {breadcrumb && breadcrumb.map((item, index) => {
+                    const name = decodeURI(item.name);
+                    return (item.path && index !== breadcrumb.length - 1 ?
                         <Link
                             key={index}
                             to={{
                                 pathname: item.path,
                                 search: `?displayType=${search.displayType}`,
                                 state: item.state,
                             }}
-                            title={item.name}
+                            title={name}
                         >
-                            {`${item.name} /`}
+                            {`${decodeURI(name)} /`}
```

Comment:
`decodeURI` is called twice on `item.name`, see this line of code above `const name = decodeURI(item.name);`